wordpress-mobile/WordPress-Android
# Do not track marking post as seen when opening it.

This PR remove tracking of marking post as seen when unseen post is opened. This is done to align stats with iOS and Calypso.

To test:
Put breakpoint here, and perform following actions:
- Open unseen post in reader. Confirm that the breakpoint is not being hit.
- Using post menu in Post Details, mark post as Seen, and then mark it as Unseen. Confirm that the breakpoint is being hit.
- From the reader post list, use post menu to mark post as Seen. Confirm that the breakpoint is being hit.

PR submission checklist:
